.appveyor.yml

```diff
@@ -0,0 +1,59 @@
+version: '{branch}.{build}'
+os: Windows
+clone_folder: C:\Projects\Cataclysm-DDA
+shallow_clone: true
+matrix:
+    fast_finish: false 
+pull_requests:
+    do_not_increment_build_number: false
+image:
+    # - Visual Studio 2017
+    - Visual Studio 2015
+configuration:
+    - Release
+    # - Debug
+platform:
+    - x64
+    # - x86
+environment:
+    matrix:
+        - TOOLCHAIN: msvc14
+install:
+    # Set TEMP folder
+    - cmd: set TEMP_FOLDER=C:\Temp
+    - cmd: mkdir "%TEMP_FOLDER%"
+    
+    # Create WINDEPEND folder
+    - cmd: mkdir "%APPVEYOR_BUILD_FOLDER%\WinDepend"
+
+    # Add WINDEPEND folder to PATH (for LUA binary)
+    - cmd: set PATH=%APPVEYOR_BUILD_FOLDER%\WinDepend;%PATH%
+
+    # Download WINDEPEND archive
+    - ps : |
+        [Net.ServicePointManager]::SecurityProtocol = 'Ssl3, Tls, Tls11, Tls12'
+        Start-FileDownload "http://dev.narc.ro/cataclysm/WinDepend-MSVC.zip" -FileName "$($env:TEMP_FOLDER)\WinDepend-MSVC.zip"
```

Comment:
@narc0tiq:

Should I change it to Dropbox to save bandwidth on your site?

Reply:
It's not a very large file, I'd say leave it like that for now and I'll check the stats in a month or so. For reference, in June I still had about 1 TB of traffic allowance left over from a 3 TB total (and the next time I decide to do maintenance that'll be a 4 TB total) -- it's not likely to be a problem unless appveyor is *really* inefficient.

By the way, I noticed you have 7zip in the deployment there, would it make more sense to use the archive at <http://dev.narc.ro/cataclysm/WinDepend-MSVC.7z>? It's almost half the size, so less likely to bump against bandwidth limits if that ends up mattering (and it should be the exact same stuff inside).

Reply:
I already forgot why I ended up with zip - probably something was not working properly, anyway I will try 7z now. Thanks.